EIPS/eip-2334.md

```diff
@@ -1,6 +1,6 @@
 ---
 eip: 2334
-title: BLS12-381 Deterministic Account Hierarchy
+title: BLS-BLS12-381 Deterministic Account Hierarchy
```

Comment:
I think this does more to confuse than to clarify. Assuming the first "BLS" is for Boneh-Lynn-Shacham, is it important to acknowledge here? The only relevant item here is the curve order.

Reply:
Agree, I thought I had removed this change, will revert

Reply:
Please see https://github.com/veorq/EIPs/commit/17242295ce1be1ad573ac7b972cf2d7b35d9e737